src/frontend/src/styles/main.css

```diff
@@ -975,11 +981,21 @@ a:hover,
  */
 
 .c-landingPage {
+  height: 100dvh;
+}
+
+.c-landingPage__container {
   display: grid;
   grid-template-columns: 1fr minmax(var(--rs-main-content-width), 1fr);
   grid-template-areas: "left right";
 
-  height: 100dvh;
+  height: 100%;
+}
+
+.c-landingPage__title {
+  font-size: calc(2.8rem + 1vw);
```

Comment:
could we use a token here? Like marketing title or something?

Reply:
Done, thanks!